src/restaurants/restaurants.js

```diff
@@ -1,23 +1,84 @@
-clickRestaurant = (e) => {
+import { str } from "ajv";
+import { getRestaurantMenu } from "../menu/menu.js";
+import { handleLocation, handleLocationByPath, route } from "../router.js";
+
+export function initializeRestaurants() {
+  getRestaurants();
+}
+
+export function getRestaurantsTemplate() {
+  return `<h1>Choose a restaurant</h1>`;
+}
+
+export function getRestaurantDetails() {
+  return `<h1>Restaurant Details</h1>`;
+}
+
+export function getRestaurantDetailsByID(id) {
   fetch(
-    `https://private-anon-af35c927af-pizzaapp.apiary-mock.com/restaurants/2`,
+    `https://private-anon-7231255228-pizzaapp.apiary-mock.com/restaurants/${id}`,
     {
       method: "GET",
     }
   )
     .then((response) => response.json())
     .then((result) => {
-      if (result == true) {
-        window.location.href = "/dist/index.html";
-        alert("You are in!");
-      } else {
-        //window.location.href = "/dist/index.html";
-        alert("Not bien...");
-      }
+      let div = document.createElement("div");
+      div.className = "main-button";
+      //debugger;
+
+      var restaurantId = document.createElement("p");
```

Comment:
Let's use only let and const.
- by default let's use `const` for declaring variables.
- `let` should be used only when that variable is changed later.

This way developers know that when they see a `let` variable it tells them it will be assigned a new value or modifier later.